bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableClient.scala

```diff
@@ -207,9 +207,9 @@ case object ConeSearchBackend extends CachedBackend with RemoteCallBackend {
   override def queryUrl(e: SearchKey): String = s"${e.url}/cgi-bin/conesearch.py"
 }
 
-case object GaiaBackend extends CachedBackend with RemoteCallBackend {
-  // For Java
-  val instance = this
+sealed trait GaiaBackend extends CachedBackend with RemoteCallBackend {
+
+  def gaia: CatalogAdapter.GaiaAdapter
```

Comment:
I assume this means fields are different between ESA/Gemini?

Reply:
It opens up the possibility of handling that situation, but it's not expected to be the case.  Instead there are two GAIA backends because of the abstract `catalog: CatalogName` in `CatalogAdapter` which must be defined differently for the two GAIA catalog servers.